.chloggen/35003-deprecate-export_created_metric.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: exporter/prometheusremotewrite
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Deprecate configuration option `export_created metric`
+
+# One or more tracking issues or pull requests related to the change
+issues: [35003]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Since the feature gate is beta in this release, we should let people know how they can re-enable the feature temporarily.
```suggestion
subtext: Disable the exporter.prometheusremotewriteexporter.deprecateCreatedMetric feature gate to temporarily re-enable the created metric.
```

---

It's the opposite, isn't it? a Beta feature gate is enabled by default, so you don't need to do anything to enable this.

Once we move to alpha then people need to manually re-enable the functionality

---

https://github.com/open-telemetry/opentelemetry-collector/tree/main/featuregate#:~:text=A%20beta%20stage%20where%20the%20feature%20has%20been%20well%20tested%20and%20is%20enabled%20by%20default%20but%20can%20be%20disabled%20through%20a%20Gate.

---

oh!  The feature gate should always progress from alpha -> beta -> stable.  Stable should mean that it is permanently disabled.  Alpha should mean that it is still enabled.

---

No worries :) I'll update the PR accordingly.